modules/enum-list/init.lua

```diff
@@ -90,7 +90,7 @@ end
 	@return {EnumItem}
 	@since v2.0.0
 ]=]
-function EnumList:GetEnumItems()
+function EnumList:GetEnumItems(): Array<EnumItem>
```

Comment:
Now that Luau has types, wouldn't {EnumItem} be more correct than Array\<EnumItem\>?